www/statusbar.js

```diff
@@ -153,16 +152,19 @@ var addStatusBarDataElement = function(){
          "StatusBar", 
          "getStatusBarHeight",
         []);
+
 }
 
 
 var injectViewportMetaTag = function(){
 
-    if(device.platform.toLowerCase().indexOf(IOS_PLATFORM) == 0 && device.version.split(".")[0] >= IOS_11_VERSION){
-        var viewportMetaElem = document.getElementsByTagName("meta").namedItem("viewport");
+    if (/(iPad)|(iPhone)/i.test(navigator.userAgent)) {
+        var version = navigator.appVersion.match(/OS (\d+)_(\d+)_?(\d+)?/);
+
+        if((version && version[1]) >= IOS_11_VERSION) {
```

Comment:
Don't you want this: if(version && (version[1] >= IOS_11_VERSION)) ??

Reply:
Also, IOS_11_VERSION is a string. Comparing integers as string can lead us to future problems.
![image](https://user-images.githubusercontent.com/15612337/37060757-2570a116-2189-11e8-8747-0c4889a01d53.png)